ts/state/selectors/conversations.ts

```diff
@@ -1064,27 +1041,20 @@ export const getMessageIsDeletable = createSelector(
 export const getMessageAttachmentProps = createSelector(getMessagePropsByMessageId, (props):
   | MessageAttachmentSelectorProps
   | undefined => {
-  if (!props || _.isEmpty(props)) {
+  if (!props || isEmpty(props)) {
     return undefined;
   }
 
-  const {
-    attachments,
-    direction,
-    isTrustedForAttachmentDownload,
-    timestamp,
-    serverTimestamp,
-    sender,
-    convoId,
-  } = props.propsForMessage;
   const msgProps: MessageAttachmentSelectorProps = {
-    attachments: attachments || [],
-    direction,
-    isTrustedForAttachmentDownload,
-    timestamp,
-    serverTimestamp,
-    sender,
-    convoId,
+    attachments: props.propsForMessage.attachments || [],
+    ...pick(props.propsForMessage, [
+      'direction',
+      'isTrustedForAttachmentDownload',
+      'timestamp',
+      'serverTimestamp',
+      'sender',
+      'convoId',
+    ]),
```

Comment:
thanks for those updates to pick syntax! :+1:

Reply:
No problem 👍